shuup/admin/modules/products/static_src/product-variation-variable-editor.js

```diff
@@ -57,10 +57,15 @@ window.VariationVariableEditor = (function(m, _) {
         ];
     }
 
+    function sortingHandle() {
+      return m("i.fa.fa-bars.sorting-handle.pull-left", "");
```

Comment:
See the indentation in the file which is 4 spaces instead of 2. This is based on our project wide decision.